src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -733,6 +749,28 @@ public function collectFromCode($content, $fileName, Module $module)
                     continue;
                 }
 
+                // Allow _t(Entity.Key, 'translation', $var) and expand _t(Entity.Key, $var)
+                if ($id == T_VARIABLE && !empty($currentEntity)) {
+                    // We have a translation, eg: _t(Entity.Key, 'translation', $var)
```

Comment:
By "we have a translation" do you mean "we've fully captured this localisation key and its default text"? If so, please change the comment to something in that vain so it's clear what you mean.

Reply:
We have a translation just means the second argument has been provided and that we can proceed

Reply:
done